config/base/default-clusterroles.yaml

```diff
@@ -34,6 +38,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: admin
+  labels:
+    app.kubernetes.io/part-of: tekton-results
```

Comment:
I think this can be removed (the metadata component adds it).

Reply:
Yes, removed it.